MediaAdminBundle/Controller/Admin/MediaController.php

```diff
@@ -135,4 +141,22 @@ public function metaAction(Request $request, $mediaId)
 
         return $this->renderAdminForm($form);
     }
+
+    /**
+     * @param string|\Symfony\Component\Form\FormTypeInterface $type
+     * @param null                                             $data
+     * @param array                                            $options
+     * @param string|null                                      $editionRole
+     * @param FolderInterface|null                             $folder
+     *
+     * @return \Symfony\Component\Form\Form
+     */
+    public function generateForm($type, $data = null, array $options = array(), $editionRole = null, $folder = null)
```

Comment:
as the first argument of your `generateForm` method are the same that the parent `createForm`, I think that you could call your method `createForm`